### PR TITLE
Add post: Adding 'Study sites'

### PR DESCRIPTION
**Pull request by simonwhatley**

### CODE REVIEW

**Comment by simonwhatley** [comment]
@Kateharries It might be a good idea to remove the link to the form to avoid spam.

**Comment by simonwhatley** [comment]
[Add ‘Study site’ section](https://trello.com/c/EhTAVb1f/1219-add-study-site-section)

**Review by darokel — APPROVED** [review]

File: app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+### Study site list
+
+On the study site list page, we show a:
+
+- button to ‘Add study site’
+- list of study sites the provider has added
+
+The list of study sites includes the:
+
+- Study site name
```
Comment (Kateharries) [inline app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md]: change to lower case 's'

File: app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+If the user has not entered a URN for a study site, we show ‘Not entered’ in the URN column.
+
+If the provider does not have any study sites, we do not show an empty list. Instead, we show the message:
+
+>A study site, such as a university campus, is where trainees do theoretical training. Add study sites for your organisation and then link one or more to a course from the ‘Basic details’ tab on the course page.
+>
+>Candidates will see a list of the selected study sites in the ‘Training locations’ section on the course page on Find.
+>
+>Find out more about adding training locations to a course.
```
Comment (Kateharries) [inline app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md]: Can we add a placeholder link here or put (link to be confirmed) after it or something? Just looks odd not having the link.

**Review by Kateharries — COMMENTED** [review]
I've left some comments on lines 68 and 79.

File: app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md (hunk 1 above)
Comment (simonwhatley) [reply app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md]: Updated

File: app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md (hunk 2 above)
Comment (simonwhatley) [reply app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md]: I've added a dummy link to example.com

**Review by victoriawinlow — COMMENTED** [review]
Looks good

**Pull request closed without merging**